.github/workflows/pr-check.yml

```
@@ -0,0 +1,32 @@
+name: Test greentea-client
+
+on: [push, pull_request]
+
+jobs:
+  astyle:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Install astyle
+      run: sudo apt-get install astyle
+
+    - name: Run astyle
+      run: |
+          git diff --name-only --diff-filter=d FETCH_HEAD..HEAD \
+            | ( grep '.*\.\(c\|cpp\|h\|hpp\)$' || true ) \
+            | while read file; do astyle -n --options=.astylerc "${file}"; done
+          git diff --exit-code --diff-filter=d --color
+
+  test-app:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Install CMake and Ninja
+      uses: lukka/get-cmake@latest
+
+    - name: Build Example Application
+      run: |
+        cmake -S example -B __build -GNinja
```

I've changed it again as per https://github.com/ARMmbed/greentea-client/pull/21#discussion_r632632398
```suggestion
        cmake -S . -B __build -GNinja
```